src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const GROUP_STYLE = sourceString.trim().split(';');
+  const GET_CLEAN_STYLE = GROUP_STYLE.map((pair) => pair.trim()).filter(
+    (style) => style !== '',
+  );
+
+  const SEPARATION_STYLE = GET_CLEAN_STYLE.map((pair) => {
+    return pair.trim().split(':');
+  });
+  const GET_STYLE_IN_OBJECT = SEPARATION_STYLE.reduce((acc, style) => {
+    acc[style[0].trim()] = style[1].trim();
```

Comment:
The code assumes that there will always be a key and a value in the style array. You should add a check to ensure that the style array has at least two elements before accessing them to avoid potential runtime errors.